### ISSUE TITLE
Freezing UI when typing to a remote client RustDesk running on RockyLinyx 9 

### ISSUE THREAD

**Issue by albgen** [rustdesk/rustdesk#3959]
### Bug Description

hello,
I have setup my own server and from windows to windows is working perfectly. Also from windows to mobile is working.
Then i tried to connect from a Windows(10 and 11) machine to a RustDesk Client running on RockyLinux 9and it is almost not usable. Basically it freezes the UI of RockyLinux and you will see what is typed after maybe 20 seconds.

### How to Reproduce

Connect from a Windows(10 or 11) Client to a RockyLinux Client. Both Clients are 1.1.9.
Rocky Linux version 9 running as virtualised on promox.

### Expected Behavior

Basically to not freeze the UI and work normally.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.1.9

### Screenshots

https://www.awesomescreenshot.com/video/16322366?key=c7f37b1e04ae407240c98549c0ea6648

### Additional Context

I see that there is a high cpu usage but it is always less than 100%. The server rust is running on docker which in turns run on the same machine Rocky where i'm conneting.

**Comment by rustdesk** [comment]
#3565
